src/misc/config.ts

```diff
@@ -2,4 +2,5 @@ export interface Config {
   apiKey: string;
   persistenceRate: number;
   sendWhatsNewMsg: boolean;
+  plugins: string[];
```

Comment:
I presume this value is used to load the plugins by name. Why does the plugins system not simply traverse a folder and load all plugins found in there? Now, the server admin has to both move plugin files to the right folder as well as explicitly name them here, instead of just moving plugin files.

Reply:
Why not. This at least forces server admins to acknowledge and understand whatever is running on their instance. I'd rather have some explicit directive via an option rather than just compiling and executing arbitrary js.